app/connectors/CalculationDetailsConnector.scala

```diff
@@ -28,13 +27,13 @@ class CalculationDetailsConnector @Inject()(httpClient: HttpClient,
                                             val appConfig: AppConfig)
                                            (implicit ec: ExecutionContext) extends IFConnector {
 
-  def getCalculationDetails(nino: String, calculationId: String)(implicit hc: HeaderCarrier): Future[CalculationDetailResponse]  = {
-    val getCalculationDetailsUrl: String = appConfig.ifBaseUrl + s"/income-tax/view/calculations/liability/$nino/$calculationId"
+  def getCalculationDetails(taxYear: String, nino: String, calculationId: String)(implicit hc: HeaderCarrier): Future[CalculationDetailResponse]  = {
```

Comment:
we might need to do some sort of verification around taxYear format;
API expecting next format => YY1-YY2

Reply:
It would be useful